static/js/ducks/candidates.js

```diff
@@ -32,12 +32,16 @@ messageHandler.add((actionType, payload, dispatch, getState) => {
   } else if (actionType === REFRESH_CANDIDATE) {
     const { candidates } = getState();
     let done = false;
-    candidates.candidates.forEach((candidate) => {
-      if (candidate.internal_key === payload.id && !done) {
-        dispatch(fetchCandidate(candidate.id, FETCH_CANDIDATE_AND_MERGE));
-        done = true;
-      }
-    });
+    if (candidates.candidates) {
+      candidates.candidates.forEach((candidate) => {
+        if (candidate.internal_key === payload.id && !done) {
+          dispatch(fetchCandidate(candidate.id, FETCH_CANDIDATE_AND_MERGE));
+          done = true;
+        }
+      });
+    } else {
+      done = true;
+    }
```

Comment:
This fixes an unrelated bug that I found during testing, when `candidates` is empty.

Reply:
```suggestion
    if (candidates.candidates) {
      candidates.candidates.forEach((candidate) => {
        if (candidate.internal_key === payload.id && !done) {
          dispatch(fetchCandidate(candidate.id, FETCH_CANDIDATE_AND_MERGE));
          done = true;
        }
      });
    } else {
      done = true;
    }
```
Let's revert this in favor of https://github.com/skyportal/skyportal/pull/1015/files#diff-06f0bdfa51b43cf02b74c11c57d522ebR35-R42  as the latter better distinguishes between null and [] values

Reply:
Done.